src/main/java/uk/gov/hmcts/probate/controller/NotificationController.java

```diff
@@ -144,7 +144,7 @@
                 log.info("Initiate call to bulk print for Caveat stopped document and coversheet for case id {} ",
                         callbackRequest.getCaseDetails().getId());
                 SendLetterResponse sendLetterResponse =
-                    bulkPrintService.sendToBulkPrintForGrant(callbackRequest, caveatRaisedDoc, coversheet);
+                    bulkPrintService.sendToBulkPrintForGrant(callbackRequest, caveatRaisedDoc, null, coversheet);
```

Comment:
Why dont you override the bulkPrintService method with sendToBulkPrintForLetterOfGrantIssued(callbackRequest, caveatRaisedDoc, coversheet)